src/Raven.Studio/typescript/viewmodels/database/tasks/transformationScriptSyntax.ts

```diff
@@ -113,26 +126,55 @@ orderData.TotalCost = Math.round(orderData.TotalCost * 100) / 100;
 // Load to SQL table 'Orders'
 loadToOrders(orderData);`;
 
-    sqlEtlSampleHtml = ko.pureComputed(() => {
-        return Prism.highlight(transformationScriptSyntax.sqlEtlSampleText, (Prism.languages as any).javascript);
-    });
+    sqlEtlSampleHtml = transformationScriptSyntax.highlightJavascript(transformationScriptSyntax.sqlEtlSampleText);
 
-    static readonly olapEtlSampleText =
+    static readonly olapEtlSamplePartitionText =
 `var orderDate = new Date(this.OrderedAt);
 var year = orderDate.getFullYear();
 var month = orderDate.getMonth() + 1;
 
-// The order of the following array values determines the partitions order in parquet file path
-loadToOrders(partitionBy([['year', year], ['month', month], ['source', $custom_field]]),
-{
+loadToOrders(partitionBy(['year', year], ['month', month]), {
+    // The order of params in the partitionBy method determines the parquet file path
     Company : this.Company,
     ShipVia : this.ShipVia
+    // Note: 2 more field are always created per table by default:
+    //       * _id: The ID column - can be overriden in the task definition
+    //       * _lastModifiedTicks: The document's last modification time column - cannot be overriden
```

Comment:
what about ticks? @aviv86 do we want to rename it and use less c# biased value?

Reply:
@ppekrol @ayende  ?